source/apiVolontaria/apiVolontaria/settings.py

```diff
@@ -144,7 +145,7 @@
         'rest_framework.permissions.IsAuthenticated',
     ),
     'DEFAULT_FILTER_BACKENDS': (
-        'rest_framework.filters.DjangoFilterBackend',
+        'django_filters.rest_framework.DjangoFilterBackend',
     ),
     'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.'
```

Comment:
Just wondering why DEFAULT_FILTER_BACKENDS has parentheses but not DEFAULT_PAGINATION_CLASS. 
The parentheses help to avoid splitting the argument string.

Reply:
`Django Rest Framework` make an iteration on each field of the list for the filter. It's not the case for the pagination.

> DEFAULT_FILTER_BACKENDS
> 
> A list of filter backend classes that should be used for generic filtering. If set to None then generic filtering is disabled.

http://www.django-rest-framework.org/api-guide/settings/#default_filter_backends

> The pagination style may be set globally, using the DEFAULT_PAGINATION_CLASS and PAGE_SIZE setting keys. For example, to use the built-in limit/offset pagination, you would do something like this:
> ```
> REST_FRAMEWORK = {
>    'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.LimitOffsetPagination',
>    'PAGE_SIZE': 100
> }
> ```

http://www.django-rest-framework.org/api-guide/pagination/#setting-the-pagination-style